site/packages/aa-core/smart-account-client/actions/sendRawUserOperation.md

```diff
@@ -0,0 +1,51 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: sendRawUserOperation
+  - - meta
+    - name: description
+      content: Overview of the sendRawUserOperation method on SmartAccountClient
+  - - meta
+    - property: og:description
+      content: Overview of the sendRawUserOperation method on SmartAccountClient
+---
+
+# sendRawUserOperation
```

Comment:
this doc already exists https://accountkit.alchemy.com/packages/aa-core/bundler-client/actions/sendRawUserOperation.html